app/routes/index.js

```diff
@@ -1,9 +1,49 @@
 const express = require('express');
 const router = express.Router();
+const bcrypt = require('bcrypt-nodejs');
+
+const usersModel = require('../database/models/usersModel');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8414579)
```suggestion
const usersModel = require("../database/models/usersModel");
```